docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md

```diff
@@ -0,0 +1,130 @@
+---
+custom_edit_url: null
+---
+
+# Askui-UI-Controller-Server
+
+
+Before running your tests you have start the Askui-UI-Controll-Server.
+There are multiple arguments which can be modified und provided to configure the Askui-UI-Contoller-Server. 
+In our implementation we use an interface which define all properties which can be provided to the Ask-UI-Contoller-Server.
+In the following we present you a short explanation for all properties.
+We also have an example to setup an controll server with specific values at the end of this page.
+
+## Table of contents
+
+### Properties
+
+- [binaryVersion](#binaryversion)
+- [display](#display)
+- [host](#host)
+- [logFilePath](#logfilepath)
+- [logLevel](#loglevel)
+- [minimize](#minimize)
+- [overWriteBinary](#overwritebinary)
+- [port](#port)
+
+## Properties
+
+### binaryVersion
+
+• **binaryVersion**: `string`
+
+There are different versions of The Askui-UI-Controller-Server.
+You can specify which binary version of server you want to download and use. The default value is 'latest' which will
+download the newest version.
+___
+
+### display
+
+• **display**: `number`
+
+
+You can choose on which display you want to excecute all
+tests. You can only perform all tests on one display.
+The application which you want to test should be open and selected on your chosen display.
+
+The default value of display is `0` which is your main monitor. If you want to use your
+second monitor you can change the value to `1` (`2` for your third monitor etc.).
+___
+
+### host
+
+• **host**: `string`
+
+
+The Host of the websocket server to connect via the runner-protocol. The default server is 127.0.0.1.
+___
+
+### logFilePath
+
+• **logFilePath**: `string`
+
+It is possible to specify a path for your log files. Log files are written to the console per default.
```

Comment:
we don't have access to the server console. when the controller is started with the lib